kong/plugin_service.go

```diff
@@ -394,6 +419,16 @@ func (s *PluginService) ListAllForRoute(ctx context.Context,
 	return s.listAllByPath(ctx, "/routes/"+*routeID+"/plugins")
 }
 
+// ListAllForConsumerGroups fetches all Plugins in Kong enabled for a consumer group.
+func (s *PluginService) ListAllForConsumerGroups(ctx context.Context,
+	cgID *string,
```

Comment:
I wonder why we want to pass the param as pointer if the `nil` value is equivalent to an empty string? If it was indicating that it's optional then somewhat this maybe would have made sense but otherwise I'm puzzled why we do this.